testcases/MachineConfig.py

```diff
@@ -325,6 +325,18 @@ def LparSetup(self):
             ioslot_drc_names = ",".join(ioslot_drc_names[:ioslot_drc_names.index("=")].split(",")[:-1]) \
                                if "=" in ioslot_drc_names else ioslot_drc_names
             self.cv_HMC.add_ioslot(ioslot_drc_names)
+        
+        if "perf=1" in self.machine_config:
+            conf = OpTestConfiguration.conf
+            if self.cv_HMC.is_perfcollection_enabled():
+                log.info("System is already booted with perf collection profile enabled")
```

Comment:
Better message will be 
"Performance Information collection is already enabled"
or 
"Performance Information Collection is active"